### DISCUSSION TITLE
Berkeley-db-1.xx keeps reporting errors when compiling

### DISCUSSION THREAD

**Discussion by leonsama** [micropython/micropython#10658] [ESP32] [answered]
Here is the log file.
[errorlog.log](https://github.com/micropython/micropython/files/10593799/errorlog.log)

I'm trying to build micropython 1.17 with ESP-IDF v4.4.3 and it seems that *Berkeley-db-1.xx* fails to compile. 

I have tried `make clean` and `git submodule update` but still doesn't work.

**Comment by leonsama** [comment, marked as answer]
solution:
1. manually remove the folder `micropython/lib/Berkeley-db-1.xx` .
2. run `git clone https://github.com/pfalcon/berkeley-db-1.xx -b embedded`  in `micropython/lib/`.

**Comment by hagibr** [comment]
This solution also helped me compiling in WSL. Thank you!
